### PR TITLE
View backup codes from account page 👀

### PR DESCRIPTION
**Pull request by pcraig3**
This PR changes things up so that we are able to view our backup codes from account page, and then generate new codes on the `/backup-codes` page.

Only when we have no backup codes do we offer to generate them from the account page.

### account page

| before | after |
|--------|-------|
|   "Get Security Codes" will generate 5 new codes     |  "View security codes" will show you your existing codes.     |
|   Screen Shot 2020-11-09 at 10 16 55 AM  | Screen Shot 2020-11-09 at 10 15 43 AM   |

### backup code page

| before | after |
|--------|-------|
|  "Visit portal" button at the bottom      |  "Get new security codes" button at the bottom     |
|  Screen Shot 2020-11-09 at 10 18 01 AM   | Screen Shot 2020-11-09 at 10 17 59 AM    |

### CODE REVIEW

File: backup_codes/templates/backup_codes/backup_codes_list.html [hunk 1]
```diff
@@ -5,7 +5,7 @@
 
 {% block content %}
   {% include  "includes/flash_messages.html" %}
-  <h1>{% trans "Security Codes" %}</h1>
+  <h1>{% trans "Security codes" %}</h1>
```
Comment (bryan-robitaille) [inline backup_codes/templates/backup_codes/backup_codes_list.html:8]: Need to add translation

File: backup_codes/templates/backup_codes/backup_codes_list.html [hunk 2]
```diff
@@ -21,6 +21,12 @@ <h1>{% trans "Security Codes" %}</h1>
 
   <p>{% trans "Write down these single-use security codes on a piece of paper." %}</p>
   <p>{% trans "Make sure to get more security codes before you use up all of the ones listed here." %}</p>
-  <div class="content-button"><a href="{% url 'start' %}" role='button' draggable='false'>{% trans "Visit portal" %}</a></div>
+  <div class="content-button">
+    <form action="{% url 'backup_codes' %}" method="post">
+      {% csrf_token %}
+      <button type="submit">{% trans "Get new security codes" %}</button>
```
Comment (bryan-robitaille) [inline backup_codes/templates/backup_codes/backup_codes_list.html:27]: Needs translation entry

File: backup_codes/templates/backup_codes/backup_codes_list.html [hunk 3]
```diff
@@ -21,6 +21,12 @@ <h1>{% trans "Security Codes" %}</h1>
...
+    </form>
+    <a href="{% url 'user_profile' pk=user.id %}" class="secondary" role='button' draggable='false'>{% trans "Back to your account" %}</a>
```
Comment (bryan-robitaille) [inline backup_codes/templates/backup_codes/backup_codes_list.html:29]: Needs translation entry

File: profiles/templates/profiles/healthcareuser_detail.html [hunk 4]
```diff
@@ -97,17 +97,23 @@ <h1>{% trans "User profile" %}</h1>
             {% switch "BACKUP_CODE" %}
             <tr>
               <th scope="row">{% trans "Security Codes" %}</th>
-              <td>{{ security_code_count }} {%trans " unused codes remaining" %}</td>
+              <td>
+                <span>
+                {{ security_code_count }}
+                {% if security_code_count == 1 %}{% trans "code" %}{% else %}{% trans "codes" %}{% endif %}
```
Comment (bryan-robitaille) [inline profiles/templates/profiles/healthcareuser_detail.html:103]: Needs translation entry

File: profiles/templates/profiles/healthcareuser_detail.html [hunk 5]
```diff
@@ -97,17 +97,23 @@ <h1>{% trans "User profile" %}</h1>
...
-
-                        {% trans "Get Security Codes" %}
-
-                    </span>
-                    <span aria-hidden="true">{% trans "Get Security Codes" %}</span>
-                  </a>
+                {% if security_code_count < 1 %}
+                  <form action="{% url 'backup_codes' %}" method="post">
+                  {% csrf_token %}
+                    <button type="submit" class="link">{% trans "Get security codes" %}</button>
```
Comment (bryan-robitaille) [inline profiles/templates/profiles/healthcareuser_detail.html:112]: Needs translation entry

File: profiles/templates/profiles/healthcareuser_detail.html [hunk 6]
```diff
@@ -97,17 +97,23 @@ <h1>{% trans "User profile" %}</h1>
...
+                  </form>
+                {% else %}
+                  <a href="{% url 'backup_codes' %}">{% trans "View security codes" %}</a>
```
Comment (bryan-robitaille) [inline profiles/templates/profiles/healthcareuser_detail.html:115]: Needs translation entry

**Review by bryan-robitaille — CHANGES_REQUESTED** [review]
Just a quick `makemessages -l fr` and `compilemessages` should do the trick to fix the internationalization issues.

**Review by bryan-robitaille — APPROVED** [review]

**Comment by bryan-robitaille** [comment]
I went ahead and fixed two entries that weren't translating on the backup_list view.  The two buttons on the page still had `fuzzy` entries.

**Comment by pcraig3** [comment]
Thanks!

**Pull request merged**